flottform/server/src/database.spec.ts

```diff
@@ -168,9 +168,68 @@ describe('Flottform database', () => {
 						session: answer,
 						iceCandidates: []
 					})
-			).rejects.toThrow(/peerkey/i);
+			).rejects.toThrow(
+				/clientKey is wrong: Another peer is already connected and you cannot change this info without the correct key anymore. If you lost your key, initiate a new Flottform connection./i
+			);
```

Comment:
Since `peerkey` isn't used in this error message but I don't see a change to the error itself, was/is this test not working? 🤔

Reply:
Yes that test was not working since as you've said the string 'peerkey' is not used anymore and is replaced by this string: 'clientKey is wrong: Another peer is already connected and you cannot change this info without the correct key anymore. If you lost your key, initiate a new Flottform connection.'
--> I fixed it in this commit: https://github.com/compose-us/flottform/pull/104/commits/481b54b79274731f0320c91e2216ec3726a87253